readme.md

```diff
@@ -1,14 +1,34 @@
 
 dockerfiles for pelias services
 
-### prerequisites
-
+### Prerequisites
+1. Minimum Docker and Docker-Compose version
 ```
 docker: min version xxx
 docker-compose: min version xxx
 ```
 
-## getting up and running
+2. In Docker > Preferences > Advanced, set the CPU to `4` and memory to `12 GB`.
```

Comment:
this step is specific to OSX, docker on linux has unlimited resourced by default.

maybe we just add a little note about this, or an OSX section if there are more OSX specific settings.